views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,82 @@
 <div id="channel-infos" ><br>
     {% if upgradeInfo.branch is not empty %}
-        <div style="clear:both">
-            <label class="label-small">{{ 'Branch:'|trans }}</label>
-            <span class="available">
-                <img src="../img/admin/{% if upgradeInfo.available is not empty %}enabled{% else %}disabled{% endif %}.gif">
-                {% if upgradeInfo.available is not empty %}
-                    {{ 'Available'|trans({}, 'Admin.Global') }}
-                {% else %}
-                    {{ 'Unavailable'|trans({}, 'Admin.Global') }}
-                {% endif %}
-            </span>
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'Branch:'|trans }}</label>
+            </div>
+            <div class="col-xs-9 col-md-11">
+                <span class="available">
+                    <img src="../img/admin/{% if upgradeInfo.available is not empty %}enabled{% else %}disabled{% endif %}.gif">
+                    {% if upgradeInfo.available is not empty %}
+                        {{ 'Available'|trans({}, 'Admin.Global') }}
+                    {% else %}
+                        {{ 'Unavailable'|trans({}, 'Admin.Global') }}
+                    {% endif %}
+                </span>
+            </div>
         </div>
     {% endif %}
     <div class="all-infos">
         {% if upgradeInfo.version_name is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Name:'|trans({}, 'Admin.Global') }}</label>
-                <span class="name">{{ upgradeInfo.version_name }}</span>
+            <div class="row">
```

Comment:
These changes are not related to the main purpose of this PR if I am not wrong?

Reply:
I need this to include a table or image for php compatibility <label> and <span> don't do the job on mobile device